sim/hunter/flanking_strike.go

```diff
@@ -107,7 +107,7 @@ func (hunter *Hunter) registerFlankingStrikeSpell() {
 		BonusCoefficient: 1,
 
 		ApplyEffects: func(sim *core.Simulation, target *core.Unit, spell *core.Spell) {
-			baseDamage := spell.Unit.MHWeaponDamage(sim, spell.MeleeAttackPower())
+			baseDamage := spell.Unit.MHWeaponDamage(sim, spell.MeleeAttackPower()) * (1 + hunter.StrikersProwessImpactMult)
```

Comment:
It looks like this set uses `Modifies Damage/Healing Done` so it should just modify the spell's `DamageMultiplier` instead of multiplying base damage